services/wallet/token/token-lists/fetcher/constants.go

```diff
@@ -1,8 +1,14 @@
 package fetcher
 
+import (
+	"fmt"
+
+	"github.com/status-im/status-go/pkg/version"
+)
+
 // TODO: update the link once we decide where to host the list of token lists.
 // remoteListOfTokenLists is the URL to fetch the list of token lists from. It needs to follow the schema defined below.
-// var remoteListOfTokenLists = fmt.Sprintf("https://raw.githubusercontent.com/status-im/status-go/refs/heads/release/%s/token-lists.json", version.Version())
+var RemoteListOfTokenLists = fmt.Sprintf("https://raw.githubusercontent.com/status-im/status-go/refs/heads/%s/_assets/wallet/token-lists.json", version.GitBranch())
```

Comment:
This location is following the branch that is being built.

Reply:
Having this in a separate repo is the best option for me. However, when we briefly discussed it today, we decided to put it where it's now for development.

> hmm but the whole point of this is to have some remote resource you can update without having to push a new release to the clients. It is by definition a reference to some changing thing and not an immutable resource.

With this approach it works exactly like that. You don't need to change the release build, it reads the list of token lists from the branch location, and status-remote list from the develop location.

Reply:
Alright, to clear up expectations and see if we're all aligned:
- Unit and Functional tests shouldn't be using this URL, downloading anything or refreshing any token list, so if we do things right there shouldn't be test flakyness coming from this.
- The client should access this to auto-update its token lists. E2E tests should ideally do it as well. We could have a special nightly unit test checking that whatever token list the URL points to is valid, parsable, etc (we've got directory `tests-unit-network` for unit tests that use the real network for this purpose, but it was never incorporated into CI)

Does this sound right?

Reply:
Sorry my comment was specifically about using referring to the current `version.GitBranch`.

I agree with everything said above 🙂